homepage/tests/appUser_model_tests.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from homepage.models import AppUser
+
+
+@pytest.mark.django_db
+class TestAppUser:
+
+    @pytest.fixture
+    def appUsers(self):
+        appUser1 = AppUser.create_appUser('user1', 'user1@mta.ac.il', 'useruser222')
+        appUser2 = AppUser.create_appUser('user2', 'user2@mta.ac.il', '1234')
+        appUser3 = AppUser.create_appUser('user3', 'user3@gmail.com', '10101010')
+
+        return (appUser1, appUser2, appUser3)
+
+    # This method tests:
+    # 1. The appUser created is the correct type of object (AppUser)
+    # 2. The appUser was created sucesfully and saved in the DB
+    def test_create_appUser(self, appUsers):

```

Comment:
Maybe you should check a failure scenario when one of the arguments passed to ```AppUser.create_appUser``` isn't valid,
and then catch the exception from ```appUser.save()```

Reply:
Added